### PR TITLE
feat(clerk-js): Retheme <Action.Card /> destructive variant

### PR DESCRIPTION
**Pull request by marcelscruz**
## Description

Added a rethemed variant for the action card.

Screenshot 2023-12-18 at 16 34 02
Screenshot 2023-12-19 at 11 38 35
Screenshot 2023-12-19 at 11 38 45

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Review by octoper — APPROVED** [review]
💯

File: packages/clerk-js/src/ui/elements/Action/ActionCard.tsx [hunk 1]
```diff
@@ -2,17 +2,20 @@ import type { ComponentProps } from 'react';
 
 import { Col } from '../../customizables';
 
-type ActionCardProps = ComponentProps<typeof Col>;
+type ActionCardProps = ComponentProps<typeof Col> & {
+  isDestructive?: boolean;
```
Comment (anagstef) [inline packages/clerk-js/src/ui/elements/Action/ActionCard.tsx:6]: For consistency, we should go with:
```suggestion
  variant?: 'neutral' | 'destructive';
```

Comment (marcelscruz) [reply packages/clerk-js/src/ui/elements/Action/ActionCard.tsx]: Updated.

**Review by desiprisg — CHANGES_REQUESTED** [review]
Let's apply the variant to org profile screens as well!

**Review by anagstef — APPROVED** [review]

**Pull request merged**